content/blog/tune-1-2-0/index.Rmd

```diff
@@ -0,0 +1,183 @@
+
+## Compute new metrics without re-fitting
+
+We also added a new function, [`compute_metrics()`](https://tune.tidymodels.org/dev/reference/compute_metrics.html), that allows for calculating metric values for metrics that were not used when evaluating against resamples. For example, consider our `xgb_res` object. Since we didn't supply any metrics to evaluate, and this model is a regression model, tidymodels selected Root Mean Squared Error and R-Squared as defaults:
+
+```{r}
+#| label: xgb-res-metrics-head
+collect_metrics(xgb_res)
+```
+
+In the past, if you wanted to evaluate that workflow against a performance metric that you hadn't included in your `tune_grid()` run, you'd need to re-run `tune_grid()`, fitting models and predicting new values all over again. Now, using the `compute_metrics()` function, you can use the `tune_grid()` output you've already generated and compute any number of new metrics without having to fit any more models as long as you use the control option `save_pred = TRUE` when tuning.
+
+So, say I want to additionally calculate Huber Loss and Mean Absolute Percent Error. I just pass those metrics along with the tuning result to `compute_metrics()`, and the result looks just like `collect_metrics()` output for the metrics originally calculated:
+
+```{r}
+#| label: compute-metrics
+compute_metrics(xgb_res, metric_set(huber_loss, mape))
+```
+
+## Easily reshape resampled metrics
+
+Finally, the `collect_metrics()` method for tune results recently [gained a new argument](https://tune.tidymodels.org/dev/reference/collect_predictions.html#arguments), `type`, indicating the shape of the returned metrics. The default, `type = "long"`, is the same shape as before. The argument value `type = "wide"` will allot each metric its own column, making it easier to compare metrics across different models.
+
+```{r}
+#| label: collect-metrics-wide
+collect_metrics(xgb_res, type = "wide")
+```
+
+Under the hood, this is indeed just a `pivot_wider()` call. We've found that it's time-consuming and error-prone to programmatically determine identifying columns when pivoting resampled metrics, so we've localized and thoroughly tested the code that we use to do so with this feature.
+
+## More love for the Brier score
+
+Tuning and resampling functions use default metrics when the user does not specify a custom metric set. For regression models, these are RMSE and R<sup>2</sup>. For classification, accuracy and the area under the ROC curve _were_ the default. We’ve also added the [Brier score](https://en.wikipedia.org/wiki/Brier_score) to the default classification metric list. 
+
+## Acknowledgements
+
```

Comment:
I think this usually gets a sentence saying thank you, rather than directly the names/github handles.